cmd/admin.go

```diff
@@ -326,6 +326,12 @@ func runCreateUser(c *cli.Context) error {
 		changePassword = c.Bool("must-change-password")
 	}
 
+	// If this is the first user being created.
+	// Take it as the admin and don't force a password update.
+	if n := models.CountUsers(); n == 0 {
+		changePassword = false
```

Comment:
May be it would be better to move it up and set this as default only so that `must-change-password` would still be used if specified